store-and-retrieve-data/use-storagehub-sdk/get-started.md

```diff
@@ -7,7 +7,14 @@ description: Set up your development environment, install the StorageHub SDK, an
 
 The StorageHub SDK is a modular toolkit that makes it easy to build on DataHaven, giving developers direct access to functionalities for managing storage, buckets, and proofs. It simplifies both on-chain and off-chain interactions so you can focus on your application logic rather than low-level integrations.
 
-This guide introduces and compares the functionalities of the StorageHub SDK packages. You'll also find prerequisites for using DataHaven and [StorageHub SDK installation instructions](#install-the-storagehub-sdk).
+This guide introduces and compares the functionalities of the StorageHub SDK packages. You'll also find prerequisites for using DataHaven and instructions for how to [install the StorageHub SDK](#install-the-storagehub-sdk), along with instructions on [how to set up all of the necessary StorageHub SDK clients](#initialize-the-storagehub-sdk) you will need to have set up in order to use the StorageHub SDK's features and the following guides.
+
+## Workflow Overview
```

Comment:
Sorry, changed my mind on this section. It's too much. In the intro let's add a way where we can link to the process, something like this:

```
You can use the StorageHub SDK for every step in the storage request and retrieval process. For more information, see the [Workflow Overview]()
```